### PR TITLE
Update marketing pages to use async calls only and fix tests

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/MarketingPageAcceptanceTests.SupplierTests/Features/AboutOrganisation/EditAboutSupplier.feature [hunk 1]
```diff
@@ -30,19 +30,4 @@ Scenario: About Supplier Section - Marked as Complete -  Any Data Saved
...
-	Given the User has created a new solution for the same supplier (Solution B)
-	And the User is editing the About supplier section for Solution B
-	When the About Supplier <field> data is changed for Solution B 
-	Then the About Supplier data is changed for Solution A as well as for Solution B
-	Examples: 
-	| field       |
-	| description |
-	| link        |
\ No newline at end of file
+	Then the About supplier section is marked as Incomplete
```
Comment (bjss-chris) [inline src/MarketingPageAcceptanceTests.SupplierTests/Features/AboutOrganisation/EditAboutSupplier.feature:33]: The old favourite: trailing new line. Do you have `insert_final_newline` set in your `.editorconfig`?

```
[*]
indent_style = space
insert_final_newline = true
```

File: src/MarketingPageAcceptanceTests.TestData/Solutions/FrameworkSolution.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿namespace MarketingPageAcceptanceTests.TestData.Solutions
+{
+    using System;
+    using System.Threading.Tasks;
+    using MarketingPageAcceptanceTests.TestData.Utils;
+
+    public class FrameworkSolution
```
Comment (bjss-chris) [inline src/MarketingPageAcceptanceTests.TestData/Solutions/FrameworkSolution.cs:7]: `sealed`?

File: src/MarketingPageAcceptanceTests.TestData/Solutions/GenerateCatalogueItem.cs [hunk 3]
```diff
@@ -35,67 +29,9 @@ public static class GenerateCatalogueItem
...
-        private static string GetTruncatedTimestamp(string prefix)
-        {
-            var timestamp = DateTimeOffset.Now.ToUnixTimeMilliseconds().ToString();
-            timestamp = timestamp[^prefix.Length..];
-
-            return prefix + timestamp;
-        }
-
-        private static string UniqueSolIdCheck(string prefix, string connectionString)
+        private static string RandomSolId(string supplierid)
```
Comment (bjss-chris) [inline src/MarketingPageAcceptanceTests.TestData/Solutions/GenerateCatalogueItem.cs:32]: Would `RandomSolutionId` be better here? (As it's not much longer in length and more obvious.)

**Review by bjss-chris — CHANGES_REQUESTED** [review]
A couple of small suggestions.

File: src/MarketingPageAcceptanceTests.SupplierTests/Features/AboutOrganisation/EditAboutSupplier.feature (hunk 1 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.SupplierTests/Features/AboutOrganisation/EditAboutSupplier.feature]: That is set in the editorconfig, but it doesn't seem to work very well on feature files

**Review by bjss-andrew — APPROVED** [review]

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**